apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -122,6 +122,11 @@ type ContourSettings struct {
 	// the annotations for Prometheus will be appended or overwritten with predefined value.
 	// +optional
 	PodAnnotations map[string]string `json:"podAnnotations,omitempty"`
+
+	// WatchNamespaces when set restricts a subset of namespaces that the contour instance
+	// will be watching.
+	// +optional
+	WatchNamespaces *string `json:"watchNamespaces,omitempty"`
```

Comment:
I'd make this API field a string slice; we can convert to a comma-separated string for the `contour serve` flag internally.

Reply:
Ok, that works